pkg/registry.go

```diff
@@ -103,4 +103,6 @@ func SetupProtocolOperations(rpcURL string, registry *ProtocolRegistry) {
 	if rocketPool != nil {
 		rocketPool.Register(registry)
 	}
+
+	registerCompoundRegistry(registry)
```

Comment:
@singhhp1069 it is being registered in this same set up function. The SupportedProtocol array enforces the usage of the `Generic` and DynamicOperation structs which is not really needed I think.